project/Dependencies.scala

```diff
@@ -312,7 +312,7 @@ object Dependencies {
 
   val UnixDomainSocket = Seq(
     libraryDependencies ++= Seq(
-      "com.github.jnr" % "jnr-unixsocket" % "0.18" // BSD/ApacheV2/CPL/MIT as per https://github.com/akka/alpakka/issues/620#issuecomment-348727265
+      "com.github.jnr" % "jnr-unixsocket" % "0.19" // BSD/ApacheV2/CPL/MIT as per https://github.com/akka/alpakka/issues/620#issuecomment-348727265
```

Comment:
What’s the fix they made? I’m wondering if they fixed a native buffer issue, in which case perhaps an alternate PR for this with consideration of using native buffers. There’s a FixMe somewhere in this code with a ref to the bug...

Reply:
Socket options weren't set correctly -- i.e. big endian was always used. This would cause buffer sizes to be quite wrong on intel arch: https://github.com/jnr/jnr-unixsocket/commit/aa2c1e43493edc9f6ca2af45a789a7bacbf196a1

https://github.com/jnr/jnr-unixsocket/issues/48 is still open though.